crates/iota-protocol-config/src/lib.rs

```diff
@@ -361,7 +363,7 @@ impl ConsensusNetwork {
 /// version.
 /// - If you want a customized getter, you can add a method in the impl.
 #[skip_serializing_none]
-#[derive(Clone, Serialize, Debug, ProtocolConfigAccessors)]
+#[derive(Clone, Serialize, Debug, ProtocolConfigAccessors, ProtocolConfigOverride)]
```

Comment:
So this proc macro is only actually used to override one field?

Reply:
All of the fields of `ProtocolConfig`

Reply:
oh the diff hid it from me